templates/greenlight/env.j2

```diff
@@ -104,7 +104,7 @@ GOOGLE_ANALYTICS_TRACKING_ID=
 # Set this to true if you want GreenLight to send verification emails upon
 # the creation of a new account
 #
-ALLOW_MAIL_NOTIFICATIONS=true
+ALLOW_MAIL_NOTIFICATIONS={{ bbb_require_email_verification }}
```

Comment:
I would rather like this variable to be similar to the name inside the config file. I also think that the current name is a it misleading, because there is `DEFAULT_REGISTRATION` for that...

Reply:
good point. I will change it to ``bbb_allow_mail_notifications``

Reply:
I updated it and squashed my changes to one commit